com.civicrm.hrjobroles/js/src/job-roles/controllers/hr-job-roles-controller.js

```diff
@@ -126,10 +126,10 @@ define([
              * @param end
              * @param {function} error
              */
-            $scope.validateDates = function (start, end, error) {
+            $scope.validateDates = function (start, end, contract_start, contract_end, error) {
```

Comment:
If i'm not wrong this function is not used anywhere in the view, so there's not reason to keep it in the `$scope` as it's basically a private function